qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/pairing_table_widget/pairing_table_widget_presenter.py

```diff
@@ -13,8 +13,8 @@
 from mantidqtinterfaces.Muon.GUI.Common.grouping_tab_widget.grouping_tab_widget_model import RowValid
 from mantidqtinterfaces.Muon.GUI.Common.grouping_table_widget.grouping_table_widget_presenter import row_colors, row_tooltips
 
-
-pair_columns = ["pair_name", "to_analyse", "group_1", "group_2", "alpha"]
+from mantidqtinterfaces.Muon.GUI.Common.pairing_table_widget.pairing_table_constants import get_pair_columns, pair_columns
+from mantidqtinterfaces.Muon.GUI.Common.utilities import table_utils
 
 
 class PairingTablePresenter(object):
```

Comment:
This class is connecting slots/signals when initialized. 
On the spirit of the MVP training, could you create a subscriber function on the view, and subscribe the presenter to it? Also, create notifier functions on the view that will call the handler's in the presenter.